docker/scripts/startup.sh

```diff
@@ -11,13 +11,17 @@ do
     esac
 done
 
+cd ..
+./google-cloud-sdk/bin/gsutil -m cp -r gs://mlcommons-data/criteo/criteo_parts/* data/
```

Comment:
we should only do this if the workload is criteo, right?

Reply:
yup removed it for now, have it locally ... will add a way to pipe dataset here so we can switch